as_perimenoun_oi_gynaikes.ass

```diff
@@ -499,21 +500,21 @@ Dialogue: 0,0:37:57.34,0:38:02.54,Default,,0,0,0,,But the women here are not afr
 Dialogue: 0,0:38:02.54,0:38:05.25,Default,,0,0,0,,Because...
 Dialogue: 0,0:38:05.25,0:38:06.62,Default,,0,0,0,,Because?
 Dialogue: 0,0:38:06.62,0:38:13.07,Default,,0,0,0,,Because Mr. Panos will drown the mother of his brother-in-law!
-Dialogue: 0,0:38:21.40,0:38:24.44,Default,,0,0,0,,I'm at odds with myself!
+Dialogue: 0,0:38:21.40,0:38:24.44,Default,,0,0,0,,I blame myself!

```

Comment:
This translation has become iconic and is not wrong, I think we are keeping "I'm at odds with myself".

Reply:
Not so strong opinion although https://dictionary.cambridge.org/dictionary/english/at-odds-with-someone-something
I think the lyricist blames himself for being still in love. He is not disagreeing with himself :)
But ok, it's poetry so yo can relax the translation

Reply:
"I am at odds" does not mean "I disagree", it means "I don't have good relations", the link you posted says that too.